libcodechecker/analyze/tidy_output_converter.py

```diff
@@ -298,6 +300,18 @@ def _create_diag(message, fmap):
         diag['path'].append(PListConverter._create_event_from_note(message,
                                                                    fmap))
 
+        hash_content = [os.path.basename(message.path),
```

Comment:
Please factor out the hash generation to one module and add comment why is it generated this way.

Reply:
Also, why did we add code to generate the hash? Didn't we already have such code? If that is not used shouldn't we delete that?